CGATPipelines/Pipeline/Execution.py

```diff
@@ -368,8 +368,8 @@ def run(**kwargs):
     options.update(kwargs.items())
 
     # insert a few legacy synonyms
-    options['cluster_options'] = options.get('job_options',
-                                             options['cluster_options'])
+    options['cluster_options'] += " " + options.get('job_options',
+                                                    "")
```

Comment:
The difference between job_options and cluster_options is the result of an unfortunate naming inconsistency between the command line parser and the Pipeline module. It might make sense to separate these semantically into job-specific and cluster options where only the latter can be over-ridden on the command line (such as setting the queue, the priorities, etc).

We need a consistent approach. Should this be the order, (later overwriting previous):

1. Hardcoded defaults
2. System cgat.ini
3. User cgat.ini
4. job_memory, job_threads, job_options specified in task
5. cluster_options specified on command line
5. job_options specified on command line